src/headers.ts

```diff
@@ -41,7 +41,15 @@ const headerValueMappers = {
     })
       .filter(Boolean).join('; ')
   },
-  permissionsPolicy: (value: PermissionsPolicyValue) => Object.entries(value).map(([directive, sources]) => (sources as string[])?.length && `${directive}=${(sources as string[]).join(' ')}`).filter(Boolean).join(', ')
+  permissionsPolicy: (value: PermissionsPolicyValue) =>
```

Comment:
issue: please change defaultConfig.ts value for permissionsPolicy from the existing one to:

```
    permissionsPolicy: {
      camera: [],
      'display-capture': [],
      fullscreen: [],
      geolocation: [],
      microphone: []
    }
```

This with the addition of the next comment would allow to have both () and your functionality :)